storybook/src/elements/nuxeo-icons/nuxeo-icons.stories.js

```diff
@@ -0,0 +1,51 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-card.js';
+import { IronMeta } from '@polymer/iron-meta/iron-meta.js';
+
+const items = new IronMeta({ type: 'iconset' }).list;
+items.forEach((item) => {
+  item.icons = item.getIconNames();
+});
+
+storiesOf('UI/nuxeo-icons', module).add(
+  'Icon catalogue',
+  () => html`
+    <style>
+      h2{
+        font-weight: 460;
+        margin-left: 10px;
+      }
+      .set {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        max-width: 1000px;
```

Comment:
What do you think of changing a hardcoded size here and instead use a percentage?